src/filters/path.rs

```diff
@@ -485,10 +485,19 @@ fn path_and_query(route: &Route) -> PathAndQuery {
 /// that the shorter path filter does not match the longer paths.
 #[macro_export]
 macro_rules! path {
-    (@start $first:tt $(/ $tail:tt)*) => ({
-        let __p = path!(@segment $first);
+        ($($pieces:tt)*) => ({
+            use $crate::internal_path;
```

Comment:
The use isn't needed, right?

Reply:
yeah, we just need to prefix all the calls with `$crate::`